### PR TITLE
clarify comments in ANIcalculator easyconfig (WIP)

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

follow up from https://github.com/easybuilders/easybuild-easyconfigs/pull/14478

### CODE REVIEW

**Comment by migueldiascosta** [comment]
marked as WIP while we explore getting around the distro dependent osdep by providing the legacy version of libnsl via a regular dependency

File: easybuild/easyconfigs/a/ANIcalculator/ANIcalculator-1.0-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -17,8 +16,13 @@ source_urls = ['https://ani.jgi.doe.gov/download_files/']
 sources = ['%(name)s_v%(version_major)s.tgz']
 checksums = ['236596a9a204cbcad162fc66be3506b2530b1f48f4f84d9647ccec3ca7483a43']
 
+# required because this is using GCCcore (for the Perl dependency)
+# even though it's not actually building anything, it's a binary package
```
Comment (jfgrimm) [inline easybuild/easyconfigs/a/ANIcalculator/ANIcalculator-1.0-GCCcore-10.3.0.eb:20]: if it's not actively needed, why not add a test exception?
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/test/easyconfigs/easyconfigs.py#L1195

Comment (migueldiascosta) [reply easybuild/easyconfigs/a/ANIcalculator/ANIcalculator-1.0-GCCcore-10.3.0.eb:20]: I think @bedroge will submit a PR with a better solution for the `libnsl` dependency, if he also addresses this `binutils` issue then this PR will be closed

Comment (bedroge) [reply easybuild/easyconfigs/a/ANIcalculator/ANIcalculator-1.0-GCCcore-10.3.0.eb:20]: Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/14494.

**Comment by migueldiascosta** [comment]
superseded by https://github.com/easybuilders/easybuild-easyconfigs/pull/14494

**Pull request closed without merging**